services/scanner.py

```
@@ -385,6 +445,180 @@ async def _guard(route):
+        """
+        # Tier 1: curl_cffi Chrome impersonation.
+        html = ""
+        try:
+            import curl_cffi.requests
+            async with curl_cffi.requests.AsyncSession(
+                impersonate="chrome120", timeout=self.timeout
+            ) as client:
+                resp = await client.get(bw_url, allow_redirects=True)
+                if resp.status_code == 200:
+                    html = resp.text or ""
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)